docs/fr/x-sendfile.md

```diff
@@ -0,0 +1,69 @@
+# Servir efficacement les gros fichiers statiques (`X-Sendfile`/`X-Accel-Redirect`)
+
+Habituellement, les fichiers statiques peuvent être servis directement par le serveur web,
+mais parfois, il est nécessaire d'exécuter du code PHP avant de les envoyer :
+contrôle d'accès, statistiques, en-têtes HTTP personnalisés...
+
+Malheureusement, utiliser PHP pour servir de gros fichiers statiques est inefficace comparé à
+à l'utilisation directe du serveur web (surcharge mémoire, diminution des performances...).
+
+FrankenPHP permet de déléguer l'envoi des fichiers statiques au serveur web
+**après** avoir exécuté du code PHP personnalisé.
+
+Pour ce faire, votre application PHP n'a qu'à définir un en-tête HTTP personnalisé
+contenant le chemin du fichier à servir. FrankenPHP se chargera du reste.
+
+Cette fonctionnalité est connue sous le nom de **`X-Sendfile`** pour Apache, et **`X-Accel-Redirect`** pour NGINX.
+
+Dans les exemples suivants, nous supposons que le "document root" du projet est le répertoire `public/`
+et que nous voulons utiliser PHP pour servir des fichiers stockés en dehors du dossier `public/`,
+depuis un répertoire nommé `private-files/`.
+
+## Configuration
+
+Tout d'abord, ajoutez la configuration suivante à votre `Caddyfile` pour activer cette fonctionnalité :
+
+```patch
+    root * public/
+    # ...
+
++	# Needed for Symfony, Laravel and other projects using the Symfony HttpFoundation component
++	request_header X-Sendfile-Type x-accel-redirect
++
++	intercept {
++		@accel header X-Accel-Redirect *
++		handle_response @accel {
++			root * private-files/
++			rewrite * {resp.header.X-Accel-Redirect}
++			method * GET
++
++			# Remove the X-Accel-Redirect header set by PHP for increased security
++			header -X-Accel-Redirect
++
```

Comment:
Should there be more headers documented in the example?

Those are the headers `BinaryFileResponse` will set.


```suggestion
+
+			header Cache-Control       {resp.header.Cache-Control}
+			header Content-Disposition {resp.header.Content-Disposition}
+			header Content-Type        {resp.header.Content-Type}
+			header Content-Length      {resp.header.Content-Length}
+			header Content-Range       {resp.header.Content-Range}
+			header Accept-Ranges       {resp.header.Accept-Ranges}
```

Reply:
Ah, never mind. I was thinking of `handle_response` in Caddy when using fastcgi, where `copy_response_headers` is needed to get all headers from upstream. In `intercept` apparently all headers are present anyway (as there's no upstream).